action.php

```diff
@@ -489,14 +489,12 @@ protected function _flattenThreads($comments, $keys = null) {
     }
 
     /**
-     * Adds a new comment and then displays all comments
+     * Checks for blocking conditions which would prevent this comment from being saved.
      *
      * @param array $comment
-     * @param string $parent
      * @return bool
      */
-    protected function _add($comment, $parent) {
-        global $ID;
+    protected function _not_spam($comment) {
```

Comment:
Could you please rename it to something that is not a double negative?  
(`_not_spam()===false` is just `isSpam()`, it isn't?)